corelib/crayfish_mesh.cpp

```diff
@@ -435,6 +436,30 @@ void Mesh::setNoProjection()
   mProjection = false;
 }
 
+void Mesh::setSourceCrsFromWKT(const QString& wkt)
+{
+    OGRSpatialReferenceH hSRS = OSRNewSpatialReference(NULL);
+
+    const char* raw_data = wkt.toUtf8().constData();
```

Comment:
this looks dangerous - wkt.toUtf8() returns a temporary QByteArray object which is deleted afterwards, so the pointer points to memory that has been freed already - that may cause crashes / misbehave